libvast/vast/bits.hpp

```diff
@@ -29,8 +29,9 @@ template <class T>
 class bits : detail::equality_comparable<bits<T>> {
   public:
   using word_type = word<T>;
-  using value_type = typename word<T>::value_type;
-  using size_type = typename word<T>::size_type;
+  using value_type = typename word_type::value_type;
+  using size_type = typename word_type::size_type;
+  static inline size_type npos = word_type::npos;
```

Comment:
Shouldn't that read `static constexpr` instead, since we're dealing with trivial integers here?

Reply:
Yep.